lib/graph/Workspace.js

```diff
@@ -74,6 +74,20 @@ class Workspace {
 		return this.#configuration.metadata.name;
 	}
 
+	/**
+	 * Returns an array of [Module]{@ui5/project/graph/Module} instances found in the configured
+	 * dependency-management resolution paths of this workspace, sorted by module ID.
+	 *
+	 * @public
+	 * @returns {Promise<@ui5/project/graph/Module[]>}
+	 *   Array of Module instances sorted by module ID
+	 */
+	async getModules() {
+		const {moduleIdMap} = await this._getResolvedModules();
+		const sortedMap = new Map([...moduleIdMap].sort((a, b) => String(a[0]).localeCompare(b[0])));
```

Comment:
No, as it will be casted to a string by localeCompare. I added the first `String()` to make sure that localeCompare can be called. But we could also omit that as we know that we only use strings as keys.